internal/controller/k0smotron.io/k0smotroncluster_controller.go

```diff
@@ -100,6 +100,11 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{Requeue: true, RequeueAfter: time.Minute}, err
 	}
 
+	if err := r.reconcileMonitoringCM(ctx, kmc); err != nil {
```

Comment:
Shouldn't we only create this if monitoring is enabled?

Reply:
Right, thanks! Fixed